requirements.txt

```diff
@@ -25,7 +25,7 @@ ldap3==2.9.1
 newrelic==8.8.0
 numpy>=1.21.0
 pandas>=1.3.0
-polars<=0.19.2
```

Comment:
did this version cause any issues?

Reply:
the `.cast` function doesn't exist in 0.17.x, and the polars documentation doesn't even have a page for under 0.18, so i just set it as a lower bound

Reply:
how about changing `<=0.19.2` to `<0.19.2`?